docs/blog/ibis-version-4.0.0-release.md

```diff
@@ -0,0 +1,41 @@
+# Ibis v4.0.0
+
+**by Patrick Clarke**
+
+09 January 2023
+
+## Introduction
+
+Ibis 4.0 has officially been released as the latest version of the package.
+This release includes several new backends, improved functionality, and some major internal refactors.
+A full list of the changes can be found in the project release notes [here](../release_notes.md).
```

Comment:
Can you avoid using "here" word in link?

> Write unique, descriptive link text that makes sense without the surrounding text. Don't use phrases such as this document, this article, or click here.


https://developers.google.com/style/link-text